### ISSUE TITLE
npm audit issue with underscore.string

### ISSUE THREAD

**Issue by cfjedimaster** [helpers/handlebars-helpers#342]
Not sure if this is an issue with *this* package to deal with or one lower, but I figured I'd start here. :) `npm audit` reveals two issues with this package related to underscore.string:

    handlebars-helpers > helper-markdown > remarkable > argparse > underscore.string

**Comment by jonschlinkert** [comment]
This should really be fixed in `argparse`. If they release a simple patch, all libraries in the ecosystem that see that message will be fixed. Otherwise, we'll need to keep to keep [hacking at the leaves](https://en.wikipedia.org/wiki/Striking_at_the_Roots).

**Comment by cfjedimaster** [comment]
From what I saw, `argparse` no longer uses it.

**Comment by cfjedimaster** [comment]
Ahah - I think it is `remarkable`. They are not using 1.0.0 of `argparse`, so the blame is with them. I think. ;)

**Comment by cfjedimaster** [comment]
And there is already a bug for it - https://github.com/jonschlinkert/remarkable/issues/321. Question is - and this is maybe meta - if remarkable refuses to fix it, does blame go to `helper-markdown`?

**Comment by jonschlinkert** [comment]
> They are not using 1.0.0 of `argparse`, so the blame is with them. I think. ;)

`argparse > underscore.string`

The problem is without a doubt with `argparse`. 

Newer versions of argparse might not be using `underscore.string`, but they should (and can easily) do a patch of an older version of argparse so that all users receive the patch. If it's patched in argparse, then it will automatically be used in all downstream libraries unless they are version locked.

**Comment by cfjedimaster** [comment]
So you would file an issue with `argparse`?

**Comment by jonschlinkert** [comment]
Yes, one would file an issue with `argparse`. If it's not fixed in argparse, then every single minor and major version of every package that uses argparse will need to release a patch.
